lib/pbench/server/database/alembic.ini

```diff
@@ -44,17 +59,17 @@ sqlalchemy.url = driver://user:pass@localhost/dbname
 # detail and examples
 
 # format using "black" - use the console_scripts runner, against the "black" entrypoint
-# hooks=black
-# black.type=console_scripts
-# black.entrypoint=black
-# black.options=-l 79
+# hooks = black
+# black.type = console_scripts
+# black.entrypoint = black
+# black.options = -l 79 REVISION_SCRIPT_FILENAME
```

Comment:
Are we sure that we want the Black support commented out?  (And, why did you bother tweaking line 65 if you were going to leave it commented out?)

Reply:
This is just an updated version of the auto-generated file.